pow/src/main/java/tech/pegasys/artemis/pow/Web3jEth1Provider.java

```diff
@@ -55,6 +58,34 @@ public Web3jEth1Provider(Web3j web3j) {
         .thenApply(EthBlock::getBlock);
   }
 
+  @Override
+  public SafeFuture<EthBlock.Block> getGuaranteedEth1BlockFuture(
+      String blockHash, AsyncRunner asyncRunner) {
```

Comment:
Is it possible to pass `AsyncRunner` into the constructor of this class rather than having to pass it into this method each time?  It should just be an implementation detail of `Web3jEth1Provider` that it delays retrying.

Reply:
Makes total sense.